CONTRIBUTING.md

```diff
@@ -25,4 +25,8 @@ information on using pull requests.
 ## Community Guidelines
 
 This project follows [Google's Open Source Community
-Guidelines](https://opensource.google.com/conduct/).
\ No newline at end of file
+Guidelines](https://opensource.google.com/conduct/).
+
+## Releasing
+
+This library is released automatically to Packagist when a [GitHub release](https://github.com/GoogleCloudPlatform/functions-framework-php/releases) is cut. To cut a release, manually create a new tag, create a release and save. The package will automatically be published to Packagist within a few minutes.
```

Comment:
```suggestion
This library is released automatically to Packagist when a
[GitHub release](https://github.com/GoogleCloudPlatform/functions-framework-php/releases)
is created. Name the release with the version appropriate for the changes 
included in that release, following semantic versioning. The package will
automatically be published to Packagist within a few minutes.
```

Reply:
Changes
 - added newlines to be consistent with the existing newline breaks in the readme (newline after 80 characters)
 - no need to create a tag first, creating the release also creates the tag
 - Adds info on naming the release according to semantic versioning

Reply:
Applied, thanks.